openwisp_ipam/admin.py

```diff
@@ -100,15 +101,30 @@ def change_view(self, request, object_id, form_url='', extra_context=None):
         }
         return super().change_view(request, object_id, form_url, extra_context)
 
+    def only_admin_view(self, view):
+        admin_site = self.admin_site
+
+        def inner(request, *args, **kwargs):
+            if not request.user.has_perm('subnet:add'):
```

Comment:
The argument passed here is wrong, it should be `openwisp_ipam.add_subnet`. This error is causing the import functionality to not work for non superusers, please add a failing test for this bug.